src/index.html

```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="
+      calendar
+      calendar--month-length--29
+      calendar--start-day--sun
+    ">
```

Comment:
Fix indentation accordingly to the code style guide. Don't split the line with class names